src/backend/distributed/executor/adaptive_executor.c

```diff
@@ -2898,6 +3087,19 @@ MillisecondsBetweenTimestamps(instr_time startTime, instr_time endTime)
 }
 
 
+/*
+ * MicrosecondsBetweenTimestamps is a helper to get the number of microseconds
+ * between timestamps when it is expected to be small enough to fit in a
+ * long.
+ */
+static long
```

Comment:
`INSTR_TIME_GET_MICROSEC` returns a `uint64` I think it makes sense to return that here too.